index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Mobile</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="header">
+      <a class="name" href="assets/images/Vahan.jpg"
+        ><p>Vahan Khachvankian</p></a
+      >
+      <div class="burger">
+        <a href="#"><img src="assets/images/lines-56763.png" alt="logo" /></a>
+      </div>
+    </div>
```

Comment:
- [ ] What a great project you have. However, It would be better to wrap this `toolbar` in a `section` tag instead of a `div`, that would improve the HTML structure and make the page more accessible 😃

Reply:
Done.